src/abstractdataframe/abstractdataframe.jl

```diff
@@ -842,11 +842,11 @@ without(df::AbstractDataFrame, c::Any) = without(df, index(df)[c])
 # catch-all to cover cases where indexing returns a DataFrame and copy doesn't
 
 Base.hcat(df::AbstractDataFrame, x; makeunique::Bool=false) =
-    hcat!(df[:, :], x, makeunique=makeunique)
+    hcat!(copy(df), x, makeunique=makeunique)
 Base.hcat(x, df::AbstractDataFrame; makeunique::Bool=false) =
-    hcat!(x, df[:, :], makeunique=makeunique)
+    hcat!(x, copy(df), makeunique=makeunique)
```

Comment:
This one wasn't and isn't needed, right?

Reply:
Good point.